pkg/driverbuilder/builder/builders.go

```diff
@@ -106,6 +113,132 @@ func Script(b Builder, c Config, kr kernelrelease.KernelRelease) (string, error)
 	return buf.String(), nil
 }
 
+type GCCVersionRequestor interface {
+	GCCVersion(kr kernelrelease.KernelRelease) float64
+}
+
+type Image struct {
+	GCCVersion []float64
+}
+
+var images = map[string]Image{
```

Comment:
Static map of images; when adding a new builder image, make sure to update this map.